app/components/candidate_interface/pending_gcse_required_component.rb

```diff
@@ -0,0 +1,26 @@
+class CandidateInterface::PendingGcseRequiredComponent < ViewComponent::Base
+  attr_accessor :application_choice, :pending_uk_gcses
+
+  def initialize(application_choice)
+    @application_choice = application_choice
+    @pending_uk_gcses = find_pending_uk_gcses(application_choice)
+  end
+
+  def pending_gcse_and_course_can_accept?
+    @pending_uk_gcses.any? && !@application_choice.course.accept_pending_gcse?
+  end
+
+private
+
+  def pending_gcse_text
+    subjects = pending_uk_gcses.map(&:subject)
+    subjects[0] = subjects[0].capitalize if subjects[0] == 'english'
```

Comment:
Can we guarantee that the first subject returned is always going to be English? Could it return ['maths', 'science', 'english'] for example. 

```
subjects = subjects.map { |subject| subject.capitalize if subject == 'english' }
```

Reply:
I whacked a `.sort_by(&:subject)` into the query method below which should sort the quals alphabetically by subject. Therefore, they should always be in the order of ['english', 'maths', 'science'] , props to @duncanjbrown for the suggestion 👍

Reply:
ah right sorry i missed that! V nice